### PR TITLE
X1079 add flag to labware table details

### PR DESCRIPTION
**Pull request by sabrine33**

### CODE REVIEW

**Comment by khelwood** [comment]
@sabrine33 Can you fix the merge conflicts in this pr please?

File: src/pages/CytAssist.tsx [hunk 1]
```diff
@@ -195,7 +195,7 @@ const SlotMappingContentTable = ({ slotCopyContent }: { slotCopyContent: SlotCop
 
       <TableBody>
         {slotCopyContent.map((slot) => (
-          <tr key={`${slot.sourceAddress}-${slot.sourceBarcode}`}>
+          <tr key={`${slot.destinationAddress}-${slot.sourceBarcode}`}>
```
[the remark this answers is not in the document]
Comment (sabrine33) [reply src/pages/CytAssist.tsx:198]: as we can create more than one labware from the same source, and that was generating duplicate key error

File: src/lib/machines/layout/layoutContext.ts [hunk 2]
```diff
@@ -25,7 +25,7 @@ export interface LayoutPlan {
 
 export interface Source {
   sampleId: number;
-  labware: LabwareFieldsFragment;
+  labware: LabwareFieldsFragment | LabwareFlaggedFieldsFragment;
```
[the remark this answers is not in the document]
Comment (sabrine33) [reply src/lib/machines/layout/layoutContext.ts:28]: there still a couple of files not included within these changes requires the LabwareFieldsFragment type

[the remark this answers is not in the document]
Comment (seenanair) [reply src/lib/machines/layout/layoutContext.ts:28]: Ok, that make sense.

**Comment by sabrine33** [comment]
Closing this and and opening a new one, as for some reason the tests are not triggered to run

**Pull request closed without merging**